DIP-36: Games of Capture and Escape (#130)
efdevcon/DIPs: DIPs/DIP-36.md

---
DIP: 36
Title: Games of Capture and Escape
Status: Proposed
Themes: Freeform
Instances: [Devcon7]
Tags: Event Production, Other
Authors: Daedalus Industries

> Discussion: Not yet posted

Reviewer: ```suggestion
Discussion: https://forum.devcon.org/t/dip-36-games-of-capture-and-escape/3447
```

// 2024-02-23 Reserving this space as a commitment to update this DIP on the side questing objective after discussion on devcon forum and with the Daedalus/SDP team. 

## Requirements

From Devcon and the devcon team, we need: 

* NPC volunteers. We will need to employ a handful of people to act as non-player characters in the game -- we would like to hook into the devcon volunteer pipeline for NPC roles within the game, and work with the volunteer coordinator on how best to organize this aspect. 
* Access and trust for players of the game to 'unlock' special places in the venue. Just as in previous years, we will mark off these special 'in-game' doors, objects, and non-player characters with an established symbol of trust. 
* A list or description of all other planned activities and points of contact for places of interest within the venue. Help us help you highlight the best aspects of the devcon experience, like the chill spot, meeting rooms, and any other programming. 

> * The core team needs elevated access permissions to the devcon teams operations (discord) so that we can set up features of the game, and more importantly so that we can prepare and prime venue staff/security about these specific locations and expected (weird) behaviors that might be mis-interpreted to be threatening without prior warning.  

Reviewer: ```suggestion
* The core team needs elevated access permissions to the devcon teams operations (discord) so that we can set up features of the game, and more importantly so that we can prepare and prime venue staff/security about these specific locations and expected (weird) behaviors that might be misinterpreted to be threatening without prior warning.  
```

* Partial funding for the team. We do these games for their own sake, but BKK is quite a commitment in time/space/effort. We ask for more than nothing, but far less than the value ultimately delivered to players. 

> # Discussion
> [put link here]
> 
> 

Reviewer: it's in the header
```suggestion
# Discussion
```
